pom.xml

```diff
@@ -100,12 +99,6 @@
             <version>${version.mockito}</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
```

Comment:
The Logback was integerted with spring-boot by default. If We remove it,  It doesn't impact the function that implements tracing associated with log for spring-boot?

Reply:
This pom is manually added, and it is the scope of the test.
It will not affect the function of the tracing associated.
I tested it.